common-rest/src/main/java/org/uniprot/api/rest/service/request/BasicRequestConverter.java

```diff
@@ -62,6 +70,32 @@ public SolrRequest.SolrRequestBuilder createStreamSolrRequest(StreamRequest requ
         return requestBuilder;
     }
 
+    public SolrRequest.SolrRequestBuilder createIdsSolrRequest(
+            SearchRequest request, String idField) {
+        SolrRequest.SolrRequestBuilder requestBuilder = createSearchSolrRequest(request);
+        requestBuilder.idField(idField);
+        updateIdsSort(requestBuilder, request.getSort());
+        return requestBuilder;
+    }
+
+    public SolrRequest.SolrRequestBuilder createIdsSolrRequest(
+            StreamRequest request, String idField) {
+        SolrRequest.SolrRequestBuilder requestBuilder = createStreamSolrRequest(request);
+        requestBuilder.idField(idField);
+        updateIdsSort(requestBuilder, request.getSort());
+        return requestBuilder;
+    }
+
+    private void updateIdsSort(SolrRequest.SolrRequestBuilder requestBuilder, String requestSorts) {
+        requestBuilder.clearSorts();
+        if (Utils.notNullNotEmpty(requestSorts)) {
+            requestBuilder.sorts(
+                    solrSortClause.getSort(requestSorts).stream()
+                            .filter(clause -> !clause.getItem().equals("score"))
+                            .toList());
+        }
+    }
+
```

Comment:
We can extract the code inside requestBuilder.sorts() to a method. And lets reuse SCORE const of AbstractSolrSortClause.
Probably, we can use equals like this !"score".equals(clause.getItem())